tests/testthat/_snaps/Printable.md

```diff
@@ -4,8 +4,8 @@
       x
     Output
       myPrintable: 
-         x: 
-         y: 
+         x: NULL 
```

Comment:
@Yuri05 This is another side effect. Values that are `NULL` were note visible when printing, now a `NULL` is shown.

Reply:
nice!